nbs/models.ipynb

```diff
@@ -57,6 +57,7 @@
     "import pandas as pd\n",
```

Comment:
<p>Since <code>quantiles</code> is a private function (only used inside classes trough <code>_calculate_intervals</code>), can we change <code>quantiles</code> to <code>_quantiles</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
:)